Migrating-AU-to-Chocolatey-AU.md

```diff
@@ -0,0 +1,22 @@
+# Migrating From `AU` To `Chocolatey-AU`
+
+With the [`1.0.0` release](https://github.com/chocolatey-community/chocolatey-au/releases/tag/1.0.0) of `Chocolatey-AU`, there is a need to migrate to the new package name.
+
+How you migrate from `AU` to `Chocolatey-AU` will depend on how you used `AU` in your repository. Below are some scenarios, but if you encounter a scenario not yet covered, please reach out in the [Chocolatey Community](https://ch0.co/community).
+
+## Your Repository Currently Clones the Original `AU` Repository
+
+If you are cloning the [original `AU` repository](https://github.com/majkinetor/au/) and building the module from there, you will want to update to clone the Chocolatey Community `Chocolatey-AU` repository: `https://github.com/chocolatey-community/chocolatey-au.git` instead. Once that has been done, you will want to [amend your update scripts](#amending-your-update-scripts) to use the `Chocolatey-AU` PowerShell Module instead of `AU`.
+
+

```

Comment:
Small nitpick. There should only be one empty line (not too important though).

Reply:
I've removed the extra line.